### DISCUSSION TITLE
blacklist /proc/1 breaks firejail

### DISCUSSION THREAD

**Discussion by sysadmin75** [netblue30/firejail#5468] [Q&A]
Deleted content

**Comment by rusty-snake** [comment]
Why do you want to run old versions?

Why do you want to blacklist `/proc/1`?

**Reply by rusty-snake** [reply]
> Why do you want to run old versions?
I'm trying to upgrade to 0.9.64.4 because that's the closest version to 0.9.62.4 that has the CVE patches for https://github.com/advisories/GHSA-m2xv-wgqg-4gxh.
Also, this issue exists in 0.9.70 as well.

OP yes. But why an old an not the latest release.

> This user has to have a very restrictive shell.

The above command is a "restricted shell" but not a "very restrictive shell" IMHO.

> So I'm trying to block access to the init process on the system

`/proc/1` is not the init process of your system. Don't blindly guess when it comes to security. 

> which we don't want the user to have access to. We don't want the user to have access to this process to prevent access to any root sensitive information on the system.

What makes the init process so special? What kind of information is exposed via init process but not via NetworkManager, cups, dnsmasq, gdm, wpa_supplicant, ... ?

To me it seems you do *something* because you *think* it is a security problem/improves security without understanding what you do and why.
